app/views/job_invocations/_form.html.erb

```diff
@@ -49,6 +49,20 @@
       </div>
     </div>
 
+    <div class="form-group">
+      <label class="col-md-2 control-label">
+        <%= _('Organization') %><br>
+        <%= _('Location') %>
+      </label>
+      <div class="col-md-4">
+        <p class="form-control-static">
+          <%= show_job_organization(Organization.current) %>
+          <br>
+          <%= show_job_location(Location.current) %>
+        </p>
+      </div>
+    </div>
+
```

Comment:
Do we want to show it in the form? Currently I don't think it adds much value, the user can always look at the org/loc switcher in the top bar.

Reply:
That's true, but sometimes you will quickly forget that you changed loc/org five minutes ago and this could be such a small "reminder" before the user hit the run button.

Reply:
I reverted it back, didn't realize that _form is also used in rerun action.